overrides/index.html

```diff
@@ -271,7 +271,7 @@ <h2 class="my-4">Verkehrsanalyse</h2>
           </video>
           <div class="service-description">
             <p>Einfach bedienbare Oberfläche</p>
-            <p>Bewegungen → Verkehrskennwerte</p>
+            <p>Bewegungen zu Verkehrskennwerten</p>
```

Comment:
I found the arrow provided more explanation than the "zu". Maybe "Bewegungen werden zu Verkehrskennwerten"